packages/communication-react/CHANGELOG.beta.md

```diff
@@ -4,99 +4,87 @@ This log was last generated on Tue, 27 Jun 2023 22:07:18 GMT and should not be m
 
 <!-- Start content -->
 
-## [1.7.0-beta.1](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.7.0-beta.1)
+## [1.7.0-beta.1](https://github.com/azure/communication-ui-library/tree/1.7.0-beta.1)
 
-Tue, 27 Jun 2023 22:07:18 GMT 
-[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.6.0...@azure/communication-react_v1.7.0-beta.1)
+Tue, 27 Jun 2023 22:07:18 GMT
+[Compare changes](https://github.com/azure/communication-ui-library/compare/1.6.0...1.7.0-beta.1)
 
-### Minor changes
+### Mention - Public Preview
+The Web UI SendBox and MessageThread components now support mentions capabilities in chat message thread. Enables user to @ mention another user by providing a Search capability to look for the desired user and then tagging them on the message.
 
-- `@azure/communication-react`
+### File Sharing
+The Web UI Chat composite now supports to view and download a file shared by a Teams user.
+
+### Optimal Video Count - Public Preview
+The Web UI Calling and CallWithChat composite now supports optimal Video Count. This provides a way to get a value which dictates the maximum number of remote renderers that should be rendered to get a good quality video.
+
+### Call Transfer - Public Preview
+The Web UI Calling composite now supports the ability to accept or reject transfer requests through CallAdapter event subscription. The Calling composite will show a transfer page when a transfer request is accepted followed by seamless transition to new call.
+
+### New CallComposite configurations to support click to call scenarios - Public Preview
+The Web UI Calling composite now supports new options to configure LocalVideoTile positioning enabling new use case scenarios with the composite like click to call
+
+### Features
+- Add the leaving page ([PR #3108](https://github.com/azure/communication-ui-library/pull/3108) by jiangnanhello@live.com)
```

Comment:
The leaving page was added to fix a bug so I would not call it feature. I would move it to Bug Fixes and change the description to 'Fix issue of intermittently showing participants when leaving while connecting to call by adding a leaving page'